### DISCUSSION TITLE
camellia-redis-proxy 在高可用/灾备中的应用场景

### DISCUSSION THREAD

**Discussion by caojiajun** [netease-im/camellia#228] [General]
camellia-redis-proxy支持双写、双读，因此可以用于一些高可用、灾备的应用场景

大家可以分享一下自己的业务场景，看看基于当前功能是否可以实现，以及是否要增强proxy的功能

**Comment by zhangyukai** [comment]
双写如果是最终一致性的话，如何确保异地集群的数据尽可能的快速的一致呢？是否有类似Redis的本地AOF以及异地Replay功能，谢谢！

**Reply by caojiajun** [reply]
camellia-redis-proxy无法保证两个集群数据是一致的，最终一致也不行，因为他只是单纯的往2个集群发送写命令，在并发场景下的乱序是可能的

**Reply by zhangyukai** [reply]
了解了，谢谢 ！
